src/retriever.jl

```diff
@@ -1,5 +1,17 @@
-module retriever
+module Retriever
 
-# package code goes here
+
+# Name all available datset scripts
+function datasets()
+    command = `retriever ls`
+    run(command)
```

Comment:
If I understand things correctly (and I'm new to Julia myself so I could totally be wrong) I think `run` prints to STDOUT and doesn't capture anything. Ideally for this function we'd capture the list of datasets so that we can work with them internally. I think we can do this using `readall` as a starting point (see more here: https://docs.julialang.org/en/release-0.4/manual/running-external-programs/).